### ISSUE TITLE
What about monthly or quarterly release strategy for minor versions?

### ISSUE THREAD

**Issue by actionless** [awesomeWM/awesome#3937]
the time shows that the current approach to release it as soon as it good enough - is not working really well, as some of small release-blockers are not getting fixed for pretty long time, and at the same time even after the past releases we were still receiving bug reports - so the code is never good enough.

it's not personally affects me in any way as an arch's AUR user, as I'm always on latest-ish git version, but it should make situation better for an average ubuntu/fedora user who are still reporting bugs for 4.3 which several years old

**Comment by Elv13** [comment]
As long as there's regressions, doing a release would break users computers. We did that in the past and it caused more harm than anything else.

**Comment by actionless** [comment]
yup, that's a problem, but how longer release period is helping to avoid that?

also that works in both directions - for bugs/regressions found in the release users wouldn't get fixes for several years

**Comment by Elv13** [comment]
I don't have enough bandwidth to fix all the bugs marked for 4.4. Someone needs to byte the bullet and submit patches

**Comment by actionless** [comment]
even if someone would - fixing and getting them merged might take 0.5 year or more (https://github.com/awesomeWM/awesome/issues?page=1&q=is%3Aopen+is%3Aissue+milestone%3Av4.4)

**Comment by Konfekt** [comment]
> fix all the bugs marked for 4.4.

@Elv13 [This list](https://github.com/awesomeWM/awesome/issues?q=sort%3Aupdated-desc%20is%3Aissue%20is%3Aopen%20milestone%3Av4.4) ? Maybe one could point it out in the README and even open bounties.

**Comment by Aire-One** [comment]
In the last few weeks, multiple distro packagers reached to us by commenting directly in some issues and by opening PRs to resolve build issues. We are now risking being pushed out of packages repositories (Debian forky/testing already removed Awesome).

Since it's unlikely that we deliver a proper release quickly, we should start to consider providing a support branch for v4.3 that backports build fixes.

**Comment by Konfekt** [comment]
Do maybe d37e3c636db1c7ff3d32e35fd69349b13c775aaa and c9f07cdb12eebf6bd2e3738424d0467dc8caaf79 cover most of what's needed to be included in Debian testing again?

> providing a support branch for v4.3 that backports build fixes.

but I suppose this is rather a question of form than substance

**Comment by actionless** [comment]
if you wanna discuss further backport-releases such as 4.3.x please make a separate topic, as this one is functionally a different thing
